packages/speech/src/index.js

```diff
@@ -239,38 +236,122 @@ Speech.findFile_ = function(file, callback) {
  *
  * @private
  *
- * @param {object} results - A `SpeechRecognitionResult` or
+ * @param {object} resultSets - A `SpeechRecognitionResult` or
  *     `StreamingRecognitionResult` object.
  * @param {boolean} verbose - Whether to use verbose mode.
  * @return {object} - The simplified results.
+ *
+ * @example
+ * var resultSets = [
```

Comment:
Take a look at this example to see the way the results are being simplified now. LMKWYT.